src/main/java/org/folio/edge/core/EdgeVerticle.java

```diff
@@ -42,6 +42,11 @@
 import io.vertx.ext.web.Router;
 import io.vertx.ext.web.RoutingContext;
 
+/**
+ * Verticle for edge module.
+ * Deprecated, use {@link EdgeVerticle2} instead.
```

Comment:
consider using the `@deprecated` tag in javadoc comment

Reply:
As Craig says. https://docs.oracle.com/javase/tutorial/java/annotations/predefined.html